cmd/login_cloud.go

```diff
@@ -20,22 +22,29 @@ import (
 //nolint:funlen,gocognit
 func getCmdLoginCloud(gs *state.GlobalState) *cobra.Command {
 	// loginCloudCommand represents the 'login cloud' command
+	var exampleText bytes.Buffer
+	exampleTemplate := template.Must(template.New("").Parse(`
+  # Show the stored token.
+  {{.}} login cloud -s
+  
+  # Store a token.
+  {{.}} login cloud -t YOUR_TOKEN
+  
+  # Log in with an email/password.
+  {{.}} login cloud`[1:]))
+
+	if err := exampleTemplate.Execute(&exampleText, gs.BinaryName); err != nil {
+		gs.Logger.WithError(err).Error("Error during help example generation")
+	}
+
```

Comment:
I feel like this can be extracted in a function that takes the template string and `gs` and returns a string that is the final example after parsing and Executing it.

Reply:
Yeah, it makes perfect sense to be honest.

Is there a file that already exists where I can create such a function? If not, where should I place that file?
On another note, should I make a test for this function? I feel like if the command output is already tested it doesn't make much sense, but YMMV, so I'd better ask.

Reply:
`cmd/common.go` seems like the correct place 

> On another note, should I make a test for this function?

I guess you can at least have 1 test, but I wouldn't go overboard on this to be honest

Reply:
Done! The naming might be a bit odd, so let me know if I should change anything.

Since I wanted to keep the function in the `cmd/common.go` as unexported, I didn't create a test for it. However, as I said, a `help` command test already exists:

https://github.com/grafana/k6/blob/9aeebf8b8e7b1017fae0f575385e03fcbf5c3394/cmd/tests/cmd_run_test.go#L80-L93

So I guess that's good enough.